### PR TITLE
Changes for Releasev5.17.0

### PR DESCRIPTION
**Pull request by chebroluharika**
### Description
Changes for Releasev5.17.0

### Issues Resolved
NA

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/_client.rb.example [hunk 1]
```diff
@@ -16,7 +16,7 @@ require 'pry'
 
 # This setting allows for use of the shortened name of the resources, like OneviewSDK::<Resource_name>,
   # when referring to OneviewSDK::<API_version>::Synergy::<Resource_name>
-# Versions supported: [200, 300, 500, 600, 800, 1000, 1200, 1600, 1800, 2000]
+# Versions supported: [200, 300, 500, 600, 800, 1000, 1200, 1600, 1800, 2000, 2200]
```
Comment (VenkateshRavula) [inline examples/_client.rb.example:19]: we are not supporting all these versions

File: README.md [hunk 2]
```diff
@@ -1,34 +1,49 @@
...
 ###  Docker Setup
-We also provide a lightweight and easy way to test and execute `oneview-sdk-ruby`. The `hpe-oneview-sdk-for-ruby:<tag>` docker image contains
-an installation of oneview-sdk-ruby and you can use it by just pulling down the Docker Image:
-The Docker Store image tag consist of two sections: <sdk_version-OV_version>
+The light weight containerized version of the HPE OneView SDK for Ruby is available in the [Docker Store](https://store.docker.com/community/images/hewlettpackardenterprise/hpe-oneview-sdk-for-ruby). The Docker Store image tag consist of two sections: <sdk_version-OV_version>
 
 ```bash
-# Download and store a local copy of oneview-sdk-ruby and
-# use it as a Docker image.
+# Download and store a local copy of oneview-sdk-ruby and use it as a Docker Image.
```
Comment (yuvirani) [inline README.md:35]: Change ov and ruby version

File: endpoints-support.md [hunk 3]
```diff
@@ -168,57 +168,57 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
...
+|<sub>/rest/logical-interconnects/{id}/compliance</sub>                                   | PUT       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/logical-interconnects/{id}/ethernetSettings</sub>                             | GET       | :heavy_multiplication_x:  |:heavy_multiplication_x:  | :heavy_multiplication_x: | :heavy_multiplication_x: |:heavy_multiplication_x: |:heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: |:heavy_multiplication_x: |:heavy_multiplication_x: |
+|<sub>/rest/logical-interconnects/{id}/ethernetSettings</sub>                             | PUT       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:198]: added twice..

File: README.md [hunk 4]
```diff
@@ -74,11 +92,12 @@ i3s_client = OneviewSDK::ImageStreamer::Client.new(
   ssl_enabled: true,                  # This is the default and strongly encouraged
   logger: Logger.new(STDOUT),         # This is the default
   log_level: :info,                   # This is the default
-  api_version: 300                    # Defaults to Min supported API version, however user should change it to max supported API version (For I3S 5.40 it is 2000)
+  api_version: 2000                    # Defaults to minimum of 300 and appliance's max API version
 )
```
Comment (yuvirani) [inline README.md:96]: Defaults to either 300 or max API version, not both. It should be appliance max version here.

File: README.md [hunk 5]
```diff
@@ -87,13 +106,13 @@ i3s_client = client.new_i3s_client(
   ssl_enabled: true,                  # This is the default and strongly encouraged
   logger: Logger.new(STDOUT),         # This is the default
   log_level: :info,                   # This is the default
-  api_version: 300                    # Defaults to Min supported API version, however user should change it to max supported API version (For I3S 5.40 it is 2000)
+  api_version: 2000                    # Defaults to minimum of 300 and appliance's max API version
```
Comment (yuvirani) [inline README.md:109]: same as above

File: README.md (hunk 2 above)
Comment (chebroluharika) [reply README.md:35]: done

File: endpoints-support.md (hunk 3 above)
Comment (chebroluharika) [reply endpoints-support.md]: removed it.

File: README.md (hunk 5 above)
Comment (chebroluharika) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (chebroluharika) [reply README.md:96]: done

File: examples/_client.rb.example (hunk 1 above)
Comment (chebroluharika) [reply examples/_client.rb.example:19]: we are having support, its just we have modified logic for defaulting API version

File: README.md [hunk 6]
```diff
@@ -471,5 +476,49 @@ First run `$ bundle` (requires the bundler gem), then...
 
 For the full testing reference please look into [TESTING.md](TESTING.md) file.
 
-## Authors
-See the [contributors graph](https://github.com/HewlettPackard/oneview-sdk-ruby/graphs/contributors)
+## Getting Help 
+
+Are you running into a road block? Have an issue with unexpected bahriov? Feel free to open a new issue on the [issue tracker](https://github.com/HewlettPackard/oneview-ansible/issues)
```
Comment (yuvirani) [inline README.md:481]: Typo "unexpected bahriov?"

Comment (chebroluharika) [reply README.md:481]: done

**Review by VenkateshRavula — APPROVED** [review]

**Review by yuvirani — APPROVED** [review]

**Pull request merged**